modules/examples/src/main/java/com/ibm/cloud/platform_services/context_based_restrictions/v1/ContextBasedRestrictionsExamples.java

```diff
@@ -68,35 +74,64 @@ public class ContextBasedRestrictionsExamples {
     protected ContextBasedRestrictionsExamples() {
     }
 
+    static {
```

Comment:
Not sure how we were actually finding and loading the config properties before, but glad that you added this :)